core/server/worker/src/main/java/alluxio/web/UIFileBlockInfo.java

```diff
@@ -0,0 +1,118 @@
+/*
```

Comment:
this and the couple of following files need to be duplicated to avoid depending on the `core-client` from the `core-server-common`